src/model/src/RUNTIME_VARS.F

```diff
@@ -609,7 +609,7 @@ SUBROUTINE INIT_ENV_VARS( JDATE, JTIME )
          CALL GET_ENV ( NLAYS_DIAG, 'NLAYS_PHOTDIAG', NLAYS_DIAG, VARDEV )
 
       ! Get Desired Wavelengths for Diagnostic Output
-        CALL GET_ENV( 'NWAVE_PHOTDIAG', NWAVE, WAVE_ENV, VARDEV )
+        CALL GET_ENV( 'NWAVE_PHOTDIAG', NWAVE, WAVE_ENV, VARDEV ) !this could be wrong, why char before NWAVE, beiming???
```

Comment:
Its possible, but need to consult with EPA.

Reply:
I have sent EPA CMAQ an issue.just check it today, now response yet.

Reply:
The only difference for RUNTIME_VARS.F is: Wei set a “cell_num” to be constant as =1. And I set it to be an integer (not limited to 1) I do this because it is a read-in value from the inputs, and should be able to change. 
The code has been tested by compile it in Orion. And it works, after changing it to be integer.